modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -131,14 +139,26 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 				suite.Require().NoError(err)
 				suite.Require().NotNil(res)
 				suite.Require().NotEqual(res.Sequence, uint64(0))
-
-				events := ctx.EventManager().Events()
-				suite.assertTransferEvents(events, coin, "memo")
```

Comment:
did this function get removed? Or is it still being used?

Reply:
This function is removed in favour of new function, which we can use everywhere

Reply:
Will it be removed in a separate pr? It is still in the [codebase](https://github.com/Anmol1696/ibc-go/blob/anmol/event-tests-transfer/modules/apps/transfer/keeper/msg_server_test.go#L11)?

Reply:
ahh sorry about that, i thought i removed it. will do asap

Reply:
Done removed it.